Gemfile

```diff
@@ -6,20 +6,21 @@ gem "rake"
 
 group :test do
   gem "benchmark-memory"
-  gem "grpc", "1.46.3"
-  gem "hiredis", "~> 0.6"
   gem "memory_profiler"
   gem "minitest"
   gem "mocha"
-  gem "mysql2", "~> 0.5"
   gem "pry-byebug", require: false
   gem "rake-compiler"
-  gem "hiredis-client", github: "redis-rb/redis-client"
-  gem "redis", github: "redis/redis-rb"
   gem "timecop"
   gem "toxiproxy"
   gem "webrick"
+
+  gem "grpc"
+  gem "mysql2", "~> 0.5"
   gem "activerecord", ">= 7.0.3"
+  gem "hiredis-client", github: "redis-rb/redis-client"
+  gem "hiredis", "~> 0.6"
+  gem "redis", github: "redis/redis-rb"
```

Comment:
For local development and run all tests need to have Gemfile with all dependencies.